### DISCUSSION TITLE
Welcome to inoERP Discussions!

### DISCUSSION THREAD

**Discussion by inoerp** [inoerp/inoERP#120] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AbbasMalik** [comment]
Hello,
I am just thinking to getting started on ERP and explore the code base. I think the new version is still under development, is it good idea to get started on the PHP version?
1. Do both of the versions share the same data model/triggers etc., or there is any difference?
2. Is there any functional difference between both of these versions?
3. Is it tested well or being used in production?
Thanks in advance.
